lib/fetch/sanitize.spec.js

```diff
@@ -504,23 +506,6 @@ describe.only('data/sanitize', () => {
         lastName: 'Nikolaievich-Savin',
       });
     });
-
-    it('converts http to https in image url', () => {
-      const rawData = {
-        id: 'badgerId',
-        data: {
-          'badger.image-url': { value: 'http://photo.png' },
-        },
-        tags: [
-        ],
-      };
-      const result = sanitizeBadger(rawData);
-      expect(result).to.deep.equal({
-        ...baseBadger,
-        id: 'badgerId',
-        primaryImageUrl: 'https://photo.png',
-      });
-    });
   });
```

Comment:
Yes, you're right. I had deleted it and after that decided to implement the fallback feature.